plugins/org.polarsys.capella.transition.system2subsystem.multiphases/src/org/polarsys/capella/transition/system2subsystem/multiphases/handlers/traceability/config/RecTraceabilityHandler.java

```diff
@@ -115,10 +115,15 @@ public void attachTraceability(EObject sourceElement_p, EObject targetElement_p,
     map.put(context_p.getSourcePhysicalArchitecture().getOwnedDataPkg(), context_p.getTempSystemAnalysis().getOwnedDataPkg());
     map.put(context_p.getSourceSystemAnalysis().getOwnedDataPkg(), context_p.getTempSystemAnalysis().getOwnedDataPkg());
     map.put(context_p.getSourceLogicalArchitecture().getOwnedDataPkg(), context_p.getTempSystemAnalysis().getOwnedDataPkg());
+    map.put(context_p.getSourceSystemEngineering().getContainedOperationalAnalysis().get(0).getOwnedDataPkg(),

```

Comment:
Some model doesn't have the OperationalAnalysis layer, or a DataPkg into it, it can raises NPE in such cases.